imod/logging/__init__.py

```diff
@@ -0,0 +1,64 @@
+"""
+Package used for providing logging support to imod-pyton.
+
+Examples
+--------
+
+If you want to directly use the logger in your project and make use of the default handlers
```

Comment:
We are providing quite some options to the user (Loguru or Python logger), where they just want something that works. Can we provide them with some defaults, which will work in ~99% of the cases?